.github/workflows/deploy-environment.yml

```diff
@@ -128,9 +128,11 @@ jobs:
           BOPS_SUBNETS: subnet-${{ secrets[format('{0}', inputs.db_subnets)] }}
           BOPS_SG: sg-${{ secrets[format('{0}', inputs.db_sg)] }}
         run: |
-          aws ecs run-task --cluster $CLUSTER \
+          task_arn=$(aws ecs run-task --cluster $CLUSTER \
             --task-definition $TASK_DEFINITION --launch-type FARGATE \
-            --network-configuration '{"awsvpcConfiguration": {"subnets": ["'$BOPS_SUBNETS'"],"securityGroups": ["'$BOPS_SG'"],"assignPublicIp": "ENABLED"}}'
+            --network-configuration '{"awsvpcConfiguration": {"subnets": ["'$BOPS_SUBNETS'"],"securityGroups": ["'$BOPS_SG'"],"assignPublicIp": "ENABLED"}}' | \
+             jq -r '.tasks[].taskArn')
+          aws ecs wait tasks-stopped --cluster $CLUSTER --tasks "$task_arn"
```

Comment:
Note that this will wait for a maximum of 10 minutes so migrations will need to run in less than that.

Also as a debugging aid maybe it's worthwhile outputting the logs using `aws logs tail`, e.g.

```
aws logs tail <log-group-name> --format short --since <start-time>
```

The start time can be generated using `start_time=$(date -Iseconds)`